Yoast/wpseo-woocommerce
# Fix grunt watch

## Context

* `grunt watch` is currently not working, this has recently been fixed in Yoast SEO and should work in every plugin.

## Summary

This PR can be summarized in the following changelog entry:

* Tooling: grunt watch (& build-watch) fix

## Relevant technical choices:

* Moved the Grunt config to the config-structure as introduced in Yoast SEO
* Added ESLint checking to Travis
* Fixed ESLint errors

## Test instructions

This PR can be tested by following these steps:

* Run `yarn`
* Run `grunt watch`
* Modify JavaScript, PHP or Grunt-configuration files
* See the appropriate checks being performed for the changed file (linting/CS for PHP, ESLint and webpack build for JS, ESLint for grunt-files)

Also introduces `build-watch`

* Run `yarn`
* Run `grunt build-watch`
* See the composer install command being executed
* See the JavaScript files being build
* See the watcher being started

Fixes #
